web-components/src/components/floating-modal/FloatingModal.stories.ts

```diff
@@ -48,11 +48,13 @@ export const FloatingModal = () => {
   const show = boolean("show", false);
   const full = boolean("full-screen", false);
   const fixed = boolean("fixed-strategy", false)
+  const minWidth = text("min-Width", "");
+  const minHeight = text("min-Height", "");
 
   return html`
 
     <md-theme class="theme-toggle" id="floating" ?darkTheme=${darkTheme} ?lumos=${lumos}>
-    <md-floating-modal ?show=${show} ?full-screen=${full} ?fixed-strategy=${fixed}>
+    <md-floating-modal ?show=${show} ?full-screen=${full} ?fixed-strategy=${fixed} .minWidth=${minWidth} .minHeight=${minHeight}>
```

Comment:
Following the comment below about kebab case, let's also make these `min-width` and `min-height`